src/spikeinterface/exporters/to_phy.py

```diff
@@ -123,7 +123,11 @@ def export_to_phy(
             rec_path = output_folder / "recording.dat"
             write_binary_recording(waveform_extractor.recording, file_paths=rec_path, dtype=dtype, **job_kwargs)
         elif isinstance(waveform_extractor.recording, BinaryRecordingExtractor):
-            rec_path = waveform_extractor.recording._kwargs["file_paths"][0]
+            if isinstance(waveform_extractor.recording, BinaryFolderRecording):
+                bin_kwargs = waveform_extractor.recording._bin_kwargs
+            else:
+                bin_kwargs = waveform_extractor.recording._kwargs
+            rec_path = bin_kwargs["file_paths"][0]
```

Comment:
This is more readable no ?
```suggestion
        elif isinstance(waveform_extractor.recording, BinaryRecordingExtractor):
            rec_path = waveform_extractor.recording._kwargs["file_paths"][0]
        elif isinstance(waveform_extractor.recording, BinaryFolderRecording):
            rec_path = waveform_extractor.recording._bin_kwargs["file_paths"][0]      
```

the `dtype = waveform_extractor.recording.get_dtype()` can be removed because handle before no ?

Reply:
I known but less readable.